xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2692,279 @@ rgw_override_bucket_index_max_shards = 12
+      realm as another zone group shares a global object namespace, ensuring
+      unique object IDs across zone groups and zones.
      </para>
+     <note>
+       <para>It is important to note that zone groups <emphasis>only</emphasis>
+       sync metadata amongst themselves. Data and metadata is replicated
+       between the zones within the zonegroup. No data or metadata is shared
+       across a realm.</para>
+     </note>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Zonegroup map</term>
+    <term>Multiple Realms</term>
     <listitem>
      <para>
-      A configuration structure that holds the map of the entire system, for
-      example which zonegroup is the master, relationships between different
-      zonegroups, and certain configuration options such as storage policies.
+      &ceph; &ogw; supports the notion of realms, which
+      can be a single zone group or multiple zone groups and a globally unique
+      namespace for the realm. Multiple realms provide the ability to support
+      numerous configurations and namespaces.
      </para>
     </listitem>
    </varlistentry>
-   <varlistentry>
-    <term>Realm</term>
-    <listitem>
-     <para>
-      A container for zonegroups. This allows for separation of zonegroups
-      between clusters. It is possible to create multiple realms, making it
-      easier to run completely different configurations in the same cluster.
-     </para>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term>Period</term>
-    <listitem>
-     <para>
-      A period holds the configuration structure for the current state of the
-      realm. Every period contains a unique ID and an epoch. Every realm has an
-      associated current period, holding the current state of configuration of
-      the zonegroups and storage policies. Any configuration change for a
-      non-master zone will increment the period's epoch. Changing the master
-      zone to a different zone will trigger the following changes:
-     </para>
-     <itemizedlist>
-      <listitem>
-       <para>
-        A new period is generated with a new period ID and epoch of 1.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Realm's current period is updated to point to the newly generated
-        period ID.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Realm's epoch is incremented.
-       </para>
-      </listitem>
-     </itemizedlist>
-    </listitem>
-   </varlistentry>
   </variablelist>
-
   <para>
-   You can configure each &ogw; to participate in a federated architecture,
-   working in an active zone configuration while allowing for writes to
-   non-master zones.
+   You can configure each &ogw; to work in an active-active zone
+   configuration, allowing for writes to non-master zones.
+   The multi-site configuration is stored within a container called a realm.
```

Comment:
Suggest introducing new terms by putting [emphasis]...[/emphasis] around them, to highlight that new words are significant and merit special attention.

Reply:
This is one thing I'm going to leave until we've finalised all the edits - but I like this suggestion. Just leaving it open so I remember to go back :)